web/modules/custom/solution/solution.module

```diff
@@ -235,38 +235,45 @@ function solution_rdf_entity_view(array &$build, EntityInterface $entity, Entity
   // @todo Add the `og-group-content` cache tag because the downloads are
   //   contained in the distribution entities which is group content.
   // @see https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-5200
+  $solution_tag = 'solution_download_count:' . md5($entity->id());
   if ($display->getComponent('download_total')) {
     $total = 0;
     if (!$entity->isNew()) {
       /** @var \Drupal\asset_distribution\AssetDistributionRelations $relationship_counselor */
       $relationship_counselor = \Drupal::service('asset_distribution.relations');
       foreach ($relationship_counselor->getSolutionDistributions($entity) as $distribution) {
-        foreach ($distribution->field_download_count->getValue() as $item) {
-          // @todo The support for fields with multiple properties has not yet
-          //   been implemented in RDF entity. These are currently returned as
-          //   multiple field items, each containing one of the properties.
-          //   Once this is fixed we no longer need to check whether the 'value'
-          //   parameter exists, it will always be there.
-          // @see https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-3576
-          // @see https://www.drupal.org/node/2892361
-          if (!empty($item['value'])) {
-            $total += (int) $item['value'];
-          }
-        }
+        $total += $distribution->download_count->entity->count->value;
       }
     }
 
-    // The '#stats_topic' property is used as the label for the statistic, and
-    // to set a template suggestion for the container.
-    // @see joinup_theme_theme_suggestions_alter()
-    // @see joinup_theme_preprocess_container__stats_topic()
-    $build['downloads'] = [
-      '#type' => 'container',
-      '#stats_topic' => t('Downloads'),
-      0 => ['#markup' => (string) $total],
-      // Hide the download stats when there are no downloads.
-      '#access' => $total > 0,
-    ];
+    if ($total > 0) {
+      // The '#stats_topic' property is used as the label for the statistic, and
+      // to set a template suggestion for the container.
+      // @see joinup_theme_theme_suggestions_alter()
+      // @see joinup_theme_preprocess_container__stats_topic()
+      $build['downloads'] = [
+        '#type' => 'container',
+        '#stats_topic' => t('Downloads'),
+        0 => ['#markup' => (string) $total],
+        // Hide the download stats when there are no downloads.
+        '#cache' => [
+          // Allow clearing the render cache of this element. By using a
+          // dedicated cache tag, we avoid invalidating the solution tag, that
+          // would result in clearing too many cache entries.
+          'tags' => [$solution_tag],
+        ],
+      ];
+    }
+    else {
```

Comment:
Yes, but this case needs a regression test to make sure we're not undoing this in the future. I'm working to provide one. Also, I think adding an outer wrapper that receives the `#cache` property would make more sense (if works).